### PR TITLE
Ws2 1778

### PR DESCRIPTION
**Pull request by Bob-E**
### Description
`.font-weight-bold` missing

### Description of problem:
`.font-weight-bold` class styles are missing after the recent Renovation theme updates.

The classes are still attached to content by Drupal, and as such, the styles should still be available. 

Discovered while testing the update with Integrated search - https://asudev.jira.com/browse/SCHWEB-1004

### Solution:
font-weight class is solved by updating to Radix 5.0.11. BS5 uses `fw-bold` instead of `font-weight-bold`.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1778)

### Checklist

- [x] Design updates match [Web Standards](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/) and [Unity Design System](https://unity.web.asu.edu)
- [x] Solution is documented on the Jira ticket
- [x] QA steps to verify have been included on the Jira ticket
- [x] No new PHP or JS errors
- [x] No accessibility issues are introduced with this update
- [x] Added/updated README.md files, if relevant

### Verified in browsers 

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] Edge

### Screenshots

Note: Sections that are not applicable for this issue can be removed.

### CODE REVIEW

**Comment by mlsamuelson** [comment]
[Visit Site](https://pr-166-webspark-ci.pantheonsite.io/)

Created multidev environment [pr-166](https://dashboard.pantheon.io/sites/aec2f75f-35eb-41f8-abe7-95b292415259#pr-166) for webspark-ci.

**Review by bebaps — CHANGES_REQUESTED** [review]
Have not yet reviewed the multi-dev. Please address these items first.

**Comment by bebaps** [comment]
@Bob-E after these two issues are resolved, please also merge in the sprint branch, and then recompile the CSS to resolve the merge conflicts. All of these things can be done in one commit. Once that's complete, it's good for a quick review and merge.

File: composer.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
         }
     ],
     "require": {
+        "drupal/radix": "^5.0",
```
[the remark this answers is not in the document]
Comment (Bob-E) [reply composer.json:49]: @bebaps Is it possible to do this as a composer command in CL like custom-require, or should I just edit the particular composer.json files directly?

[the remark this answers is not in the document]
Comment (bebaps) [reply composer.json]: @Bob-E in this case, just edit the proper composer file directly.

File: web/themes/webspark/renovation/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
         "sass-loader": "^12.1.0"
     },
     "dependencies": {
-        "@asu/unity-bootstrap-theme": "1.7.0",
+        "@asu/unity-bootstrap-theme": "1.7.1",
```
[the remark this answers is not in the document]
Comment (bebaps) [reply web/themes/webspark/renovation/package.json]: @Bob-E that is not needed, unless it is within the scope of the ticket. In this case, it's fine to leave it on 1.7.0.

**Comment by Bob-E** [comment]
@bebaps I merged ws2-1778 and ws2-sprint-46, resolved merge conflicts, recompiled the Renovation, and made a new commit. Does it look good? There were quite a few file changes with the merge, but I assume that's normal.

**Review by bebaps — APPROVED** [review]
Approved.

**Pull request merged**